avatica/src/main/java/org/apache/calcite/avatica/proto/Requests.java

```diff
@@ -14,7 +14,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 // Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Oops, my bad, it was on a JIRA issue: https://issues.apache.org/jira/browse/CALCITE-840

I'd have to look into it, but last I checked, the available plugins just exec'ed out to the command line. In other words, you'd still have to have the compiler installed, you just wouldn't need to invoke things yourself.